ProjectManagmentSystem/src/Modules/ProjectsModule/components/ProjectsList/ProjectsList.tsx

```diff
@@ -9,46 +9,86 @@ import Modal from "react-bootstrap/Modal";
 import DeleteData from "./../../../SharedModule/components/DeleteData/DeleteData";
 import { useToast } from "../../../Context/ToastContext";
 import Loading from "../../../SharedModule/components/Loading/Loading";
+
+
+
+
 export default function ProjectsList() {
   const { requestHeaders, baseUrl }: any = useContext(AuthContext);
   const { getToast } = useToast();
   const [projectsList, setProjectsList] = useState([]);
   const [ProjectId, setProjectId] = useState(0);
   const [isLoading, setIsLoading] = useState(false);
-
+  const [totalResults, setTotalResults] = useState(0);
```

Comment:
We might consider adding a pagination component, as we will need it in multiple pages. This will make the pagination consistent across all pages.

Reply:
Done.